pages/events/index.vue

```diff
@@ -349,6 +352,19 @@ export default class EventsPage extends QiskitPage {
         max-width: 24rem;
       }
     }
+
+    a {
```

Comment:
We try to avoid styling html tags, could you use classes instead?

Reply:
of course, I also got rid of the event-page__anchor class since I wasn't really using it